src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,16 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+
 public class Application {
+    private static final int BALLS_LENGTH = 3;
+
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Loterry loterry = new Loterry();
+        Ball[] balls = new Ball[BALLS_LENGTH];
+        for (int i = 0; i < balls.length; i++) {
+            balls[i] = loterry.getRandomBall();
+        }
+        System.out.println(Arrays.toString(balls));
```

Comment:
rewrite this without using an array - just call get random ball method inside of a loop